pacmacs-image.el

```diff
@@ -65,20 +75,39 @@
                            (if bit 1 0))))
     result))
 
-(defun pacmacs--bits-to-xpm (bits width height)
+(defun pacmacs--generate-xpm-palette (palette)
+  (let* ((n (length palette))
+         (palette-indices (number-sequence 0 (1- n))))
+    (->> (-map #'identity palette)
```

Comment:
Hm... palette is a list everywhere now. I believe I can get rid of that conversion.